tsfm_public/toolkit/get_model.py

```diff
@@ -0,0 +1,103 @@
+import os
+
+import yaml
+
+from tsfm_public.models.tinytimemixer import TinyTimeMixerForPrediction
+
+
+def check_ttm_model_path(model_path):
+    if "ibm-granite/granite-timeseries-ttm-r1" in model_path:
+        return 1
+    elif "ibm-granite/granite-timeseries-ttm-r2" in model_path:
+        return 2
+    elif "ibm/ttm-research-r2" in model_path:
+        return 3
+    else:
+        return 0
+
+
+class GetTTM(TinyTimeMixerForPrediction):
+    @classmethod
+    def from_pretrained(cls, model_path, context_length=None, forecast_length=None, **kwargs):
```

Comment:
Can we try use "prediction" exclusively

Reply:
@wgifford - you mean `prediction_length` instead of `forecast_length` ?

Reply:
Yes

Reply:
Done